docs/getting-started/mobile/ios/index.md

```diff
@@ -4,8 +4,92 @@ sidebar_position: 1
 
 # iOS
```

Comment:
❓ Is there not an Android version of this guide available?

Reply:
It's but we discussed it in Slack and it's going to be handled in a different PR by @patrick-bitwarden so we keep author attribution for the docs.